images/go/configs/latest-1.20.apko.yaml

```diff
@@ -0,0 +1,19 @@
+contents:
```

Comment:
Maybe just 1.20.apko.yaml? We don't need the latest part I think.

Reply:
Fixed!